projects/RabbitMQ.Client/client/api/PublicationAddress.cs

```diff
@@ -116,6 +116,23 @@ public static PublicationAddress Parse(string uriLikeString)
             return null;
         }
 
+        public static PublicationAddress TryParse(string uriLikeString)
```

Comment:
TryParse methods usually have a signature like `bool TryParse(string uriLikeString, out PublicationAddress result)`

Where the return value indicates parsing success.